youtube_dl/extractor/ciscolive.py

```diff
@@ -65,45 +82,49 @@ def _parse_rf_item(self, rf_item):
 
 
 class CiscoLiveSessionIE(CiscoLiveBaseIE):
-    _VALID_URL = r'https?://(?:www\.)?ciscolive(?:\.cisco)?\.com/[^#]*#/session/(?P<id>[^/?&]+)'
+    _VALID_URL = r'https?://(?:www\.)?ciscolive\.com/[^#]*#/session/(?P<id>[^/?&]+)'
```

Comment:
The extractor doesn't visit the URL, does it? With the original pattern, would someone be able to download a session using a saved URL in the old format? If so, I suggest keeping the original. Or do they expire so that a saved URL would only be transiently useful?

Reply:
You are correct the extractor does not visit the URL, the regex extracts the id of the session which is then used against the ciscolive session api.

A saved URL seems to have a lifetime of a few years.
This is based on the session id's being unavailable (removed) at some stage (which only appears to be due to the content being removed from the ciscolive session api).

It seems reasonable to continue to allow the old format as its plausible some older session id's (using the old format) may still be available.
No new links have been provided with the ciscolive.cisco.com domain for a few years now, so the value this compatibility may have in the future will likely diminish.

I have reverted this change.